opentimelineio_contrib/adapters/aaf_adapter/aaf_writer.py

```diff
@@ -217,10 +217,10 @@ def _from_aaf_file(clip):
         mob_id = None
         target_url = clip.media_reference.target_url
         if os.path.isfile(target_url) and target_url.endswith("aaf"):
-            aaf_file = aaf2.open(clip.media_reference.target_url)
-            mastermobs = list(aaf_file.content.mastermobs())
-            if len(mastermobs) == 1:
-                mob_id = mastermobs[0].mob_id
+            with aaf2.open(clip.media_reference.target_url) as aaf_file:
+                mastermobs = list(aaf_file.content.mastermobs())
+                if len(mastermobs) == 1:
+                    mob_id = mastermobs[0].mob_id
         return mob_id
```

Comment:
FYI, switching to this "with ..." syntax avoids holding the file open. I was seeing some warnings about unclosed file handles when running the tests.